### ISSUE TITLE
Successful authentication sign out player immediately if load achievements fails

### ISSUE THREAD

**Issue by DiegoDePalacio** [playgameservices/play-games-plugin-for-unity#3028]
In the [process of Authentication](https://github.com/playgameservices/play-games-plugin-for-unity/blob/728e560bb985ba6c007eebb99c005615942f8096/source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs#L70), a [callback notifying a successful sign-in is called](https://github.com/playgameservices/play-games-plugin-for-unity/blob/728e560bb985ba6c007eebb99c005615942f8096/source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs#L148), but [sign out](https://github.com/playgameservices/play-games-plugin-for-unity/blob/728e560bb985ba6c007eebb99c005615942f8096/source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs#L795) the player [silently](https://github.com/playgameservices/play-games-plugin-for-unity/blob/728e560bb985ba6c007eebb99c005615942f8096/source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs#L150) if [LoadAchievements ](https://github.com/playgameservices/play-games-plugin-for-unity/blob/728e560bb985ba6c007eebb99c005615942f8096/source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs#L746)is not possible.

This leads to unexpected situations on the app or game using the GooglePlayGames' AndroidClient.

IMHO, the authentication process should also have a callback to notify the app that loading achievements was not possible and maybe even allow to leave the player signed in, even in the case the LoadAchievements failed, if such achievements are not going to be used at all by the app.

**Comment by nickfourtimes** [comment]
I'm having the same issue. I didn't have any achievements defined for my game, and I was getting this error; I tried defining a single test achievement and trying again, but the error persisted.

The only solution I've found so far is commenting out L150 of [AndroidClient.cs](https://github.com/playgameservices/play-games-plugin-for-unity/blob/728e560bb985ba6c007eebb99c005615942f8096/source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs):
`LoadAchievements(ignore => { });`

Once this is done, my test user is no longer automatically signed out when LoadAchievements comes up against the `CLIENT_RECONNECT_REQUIRED` error. However, I'm not sure if that's a sustainable solution, if & when I want to add achievements into my game.

**Comment by mohumohu-corp** [comment]
I'm in the exact same situation.  I do not have a single Achivements set up, but I get this error.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
